packages/vitest/src/node/index.ts

```diff
@@ -8,3 +8,5 @@ export function defineConfig(config: UserConfig) {
 export type { Vitest } from './core'
 export { createVitest } from './create'
 export { VitestPlugin } from './plugins'
+export { VitestRunner, getVitestRunnerOptions } from './execute'
```

Comment:
this actually should be run in `runtime`, but they are inside `node`

should I move it?